lib/pbench/server/database/models/datasets.py

```diff
@@ -723,8 +745,11 @@ class Metadata(Database.Base):
 
     # --- Standard Metadata keys
 
+    # Metadata keys that clients can update
+    USER_UPDATEABLE_METADATA = [SAVED, SEEN, USER]
+
     # Metadata keys that are accessible to clients
-    USER_METADATA = [DELETION, SAVED, SEEN]
+    USER_METADATA = USER_UPDATEABLE_METADATA + [DELETION]
```

Comment:
Why are `"access"` and `"owner"` added to this list later?  (I.e., why isn't that `METADATA` definition composed here?)

Reply:
`deletion` is a Metadata key that's not directly client-modifiable. `access` and `owner` are `Dataset` columns, not `Metadata`; treating them as part of the "metadata" namespace is therefore an artifact of the API implementation, not this module.

Reply:
OK.  😒  This "smells" like we don't have the right solution, somehow.